dist/index.html

```diff
@@ -74,7 +74,7 @@
 
   // Build a system
   const ui = SwaggerUIBundle({
-    url: "https://lapp.ap.ngrok.io/doc/swagger/",
+    url: "{{ swagger_yml_path }}",
```

Comment:
- Didn't see where this value come from both from the docker-compose yml or the toml config in the corresponding lapp PR.
- Can provide a default value so that it can run in the docker independently?